### PR TITLE
Part5

### PR DESCRIPTION
**Pull request by dantevibes**
Finished part 5: added functions with For loops doing operations on member variables. Called functions from main. added cout statements to print what is going on inside for loops.Some return values mid-loop.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -157,6 +159,20 @@ void Octopus::squirtInk(float inkSize, char escapeDirection)
     std::cout << "Ink amount after squirt: " << inkAmount <<std::endl;
 }
 
+float Octopus::huntForFood(int howHungry)
+{
+    for(int i = 0 ; i < howHungry ; ++i )
+    {
+        catchAFish(i, i) ? size += 2.2f : size -= 1.1f ;
```
Comment (matkatmusic) [inline main.cpp]: Don't write code like this. 
if `catchAFish()` returns something, then use a correct `if()` to show that. 
```
if( catchAFish(i, i) != 0 )
    size += 2.2f;
else
    size -= 1.1f;
```
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -213,7 +231,21 @@ void Band::postOnInsta(int numOfPics, char filter, std::string postText)
...
+    std::cout << "Insta text: " << postText << std::endl;
+}
+
+double Band::goOnTour(int numOfDestinations, double audience, bool isGrowing)
+{
+    numOfSongs = 15;
+    budget = 5000.0;
+    for(int i = 0 ; i < numOfDestinations ; ++i )
+    {
+        (isGrowing) ? audience *= 2.22 : audience /= 2.22;
```
Comment (matkatmusic) [inline main.cpp]: This is a bad coding style to develop. 
you should either use `if( isGrowing ) ...`
or
```
audience = (isGrowing) ? audience * 2.22 : audience / 2.22;
```
Always strive to make your code as easy to understand as possible
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (dantevibes) [reply main.cpp]: is it okay to do it as
size = catchAFish(i,i) ? size +2.2f : size - 1.1f ;
As you posted in the next comment? When/how do I use ternary operators properly?

Comment (dantevibes) [reply main.cpp:166]: Of I see...use an If because catchafish returns a bool

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**